### PR TITLE
add librsvg-2

### PR DESCRIPTION
**Pull request by deviator**

### CODE REVIEW

**Comment by deviator** [comment]
fix #192

**Comment by deviator** [comment]
ping @MikeWey 
Have I done everything right? Autogenerated files needs to add for windows users? Because they have no gir files and they can't run wrapper, right?

File: src/APILookupRsvg.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+addAliases: start
+	public import gtkc.cairotypes;
+	public import gtkc.glibtypes;
+	public import gtkc.gobjecttypes;
+        public import gtkc.gdkpixbuftypes;
+
+addAliases: end
+
+struct: Handle
+class: Handle
```
Comment (MikeWey) [inline src/APILookupRsvg.txt]: These two lines (34, 35) are not needed.

**Comment by MikeWey** [comment]
> Autogenerated files needs to add for windows users? Because they have no gir files and they can't run wrapper, right?

Yes, for now the generated code needs to be included. Getting things to a point where it isn't needed is a work in progress.

**Comment by deviator** [comment]
> The updated version of generated/gtkd/gtkc/paths.d is missing.
On this version it at gtkd/gtkd path. And I have some question about generation: some functions was not found (in other APILookup files), I remove they for working wrapper, and before some classes was not complete (some structs are not defined, functions body are missing). It's normal? Can this be due to mismatch of gtk versions on my and your systems?

**Comment by MikeWey** [comment]
You probably have an older version of Gtk+ installed on your system. The generator currently doesn't handle that al that well, which is one of the reasons the generated source is in the repo.

**Pull request merged**